GitCommands/Settings/AvatarFallbackType.cs

```diff
@@ -20,14 +19,11 @@ public enum GravatarFallbackAvatarType
         [Description("Author initials")]
         AuthorInitials,
 
-#if false
         /// <summary>
-        /// Return a simple, cartoon-style silhouetted outline of a person (provided by Gravatar service)
+        /// Return a simple, cartoon-style silhouetted outline of a person
         /// (does not vary by email hash)
         /// </summary>
-        /// Don't use it, use Resources.User image instead
         MysteryMan,
-#endif
```

Comment:
It was "essentially" deleted because it was replaced with the Resources.User image (according to the comment) my guess is to prevent network calls for images that will never change and instead use the embedded one. Because the current implementation doesn't make network calls for the "MysteryMan" Type I reenabled it.
I can also delete it, your call.

Reply:
We can have a comment about it, but since we don't use let's just delete it. Less code - the better :)

Reply:
✔ done